### PR TITLE
Fix V5

### PR DESCRIPTION
**Pull request by hwangsihu**

### CODE REVIEW

**Comment by LucasB25** [comment]
update pr

**Comment by hwangsihu** [comment]
What do you mean

**Comment by LucasB25** [comment]
image

**Review by appujet — CHANGES_REQUESTED** [review]
Don't need to change to BETA 

Beta will be Marge in few days

**Review by appujet — CHANGES_REQUESTED** [review]
Why you removed ?

**Comment by hwangsihu** [comment]
Oh no, I was trying to run two bots to differentiate them, but I accidentally pushed it to that one. I'll run it back after I update the translation.
As for removing the command, as I said in the commit description, I don't know if it's due to the influence of lavalink-client, but I don't think it's necessary because it seems like the pitch and speed are saved. We can just make the pitch and speed the same.

**Comment by hwangsihu** [comment]
Okay, I added it again. It's definitely annoying without it LOL

**Review by appujet — CHANGES_REQUESTED** [review]

**Review by appujet — CHANGES_REQUESTED** [review]
Why did you enable Spotify, Apple Music, and other sources but set them to `false` by default in the `application.yml`?

**Comment by LucasB25** [comment]
Why would you want to add the lyrics plugin which does exactly the same thing as lavasrc lyrics?

**Comment by hwangsihu** [comment]
The developer on Discord asked me to install that plugin, otherwise you could uninstall it yourself, I can't get it to work right now.

**Comment by hwangsihu** [comment]
https://discord.com/channels/1082302532421943407/1082302533332115539/1285126518825029673

**Comment by infnibor** [comment]
> https://discord.com/channels/1082302532421943407/1082302533332115539/1285126518825029673

"LavaSrc has support for said plugins
But it does not ship them
You need to install them by yourself if you want these features"

Read this again, just like lava-search are supported but you have to import the plugin from somewhere, the operation of which you set in lavasrc etc. The fact that you set the lyrics sources to on in lavasrc will only throw errors, because lavasrc will not find the necessary data / variables, which will be added by adding the lavalyrics plugin

**Comment by LucasB25** [comment]
> > https://discord.com/channels/1082302532421943407/1082302533332115539/1285126518825029673
> 
> "LavaSrc has support for said plugins But it does not ship them You need to install them by yourself if you want these features"
> 
> Read this again, just like lava-search are supported but you have to import the plugin from somewhere, the operation of which you set in lavasrc etc. The fact that you set the lyrics sources to on in lavasrc will only throw errors, because lavasrc will not find the necessary data / variables, which will be added by adding the lavalyrics plugin

https://discord.com/channels/1082302532421943407/1082302533332115539/1285492495735848990

**Comment by hwangsihu** [comment]
```
npm start

> lavamusic@5.0.0-beta start
> npm run clean && node .


> lavamusic@5.0.0-beta clean
> node scripts/clean.js && npm run build


> lavamusic@5.0.0-beta build
> tsc --project tsconfig.json

src/utils/functions/player.ts:94:9 - error TS2367: This comparison appears to be unintentional because the types '"soundcloud" | "deezer" | "bandcamp" | "twitch" | "applemusic" | "yandexmusic" | "flowery-tts"' and '"jiosaavn"' have no overlap.

94     if (lastTrack.info.sourceName === "jiosaavn") {
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in src/utils/functions/player.ts:94
```
Hm

File: src/utils/functions/player.ts [hunk 1]
```diff
@@ -91,7 +91,7 @@ export async function autoPlayFunction(player: Player, lastTrack?: Track): Promi
             );
         return;
     }
-    if (lastTrack.info.sourceName === "jiosaavn") {
```
Comment (appujet) [inline src/utils/functions/player.ts:94]: What version are you using in lavalink client ?

Comment (appujet) [reply src/utils/functions/player.ts:94]: There doesn't need to do `any type`

**Review by appujet — APPROVED** [review]

**Comment by hwangsihu** [comment]
But how do you fix player lag? It was fine in V4.

**Comment by hwangsihu** [comment]
> What version are you using in lavalink client ?

Check out my package.json, I'm using 2.3.6, but not work, so I fixed it.

**Comment by LucasB25** [comment]
Please recreate a new PR because I made changes to your PR that you overwrote without having retrieved them before...

File: src/utils/functions/player.ts (hunk 1 above)
Comment (appujet) [reply src/utils/functions/player.ts]: Please revert this commit

**Pull request closed without merging**